public/css/freelancer.css

```diff
@@ -219,14 +231,171 @@ ul.tag-list li {
   display: none;
 }
 
-.bg-orange {
+#freelancer-claim.bg-orange {
   background-color: orange;
+  padding-right: 20px;
+  padding-left: 20px;
+  color: var(--basic-white);
 }
 
 .bg-green {
-  background-color: green;
+  background-color: #79b1ed;
+}
+
+.bg-green #freelancer-claim-status {
+  height: 30px;
+  color: var(--basic-white);
 }
 
 .bg-yellow {
   background-color: yellow;
```

Comment:
Here you don't set the color property.
In general, instead of adding on all different backgrounds `color: var(--basic-white);` you could use the id of the container `#freelancer-claim`.